addons/point_of_sale/static/src/xml/Popups/CashMoveReceipt.xml

```diff
@@ -64,6 +64,9 @@
                 <span t-esc="_receipt.reason" class="pos-receipt-right-align" />
             </div>
             <br />
+            <div class="pos-receipt-order-data">
+                <div><t t-esc="_receipt.date" /></div>
```

Comment:
We can get rid of the wrapper `div`, right? If so, kindly remove it and then we're good to go.